__tests__/helpers/database.js

```diff
@@ -0,0 +1,36 @@
+import {
```

Comment:
why database? The code below is using orm, which is part of the application. It's not database itself

Reply:
because its database helper :) 
do you think orm helper would be better?

Reply:
it's factories
check out https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md

Reply:
there are getters as well

Reply:
like find.user(..)

Reply:
check out the comment below
https://github.com/alex-popov-tech/backend-project-lvl4/pull/50#discussion_r614559676